### PR TITLE
#3184 Load list data into the toolbox

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3184)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "setup": "yarn && yarn build",
     "start": "react-scripts start",
-    "build": "cross-env CI=true react-scripts build",
+    "build": "react-scripts build",
```
Comment (konekoya) [inline ohara-manager/client/package.json]: @wu87988622 I think this check is not necessary.`yarn lint` will catch linting errors anyway

File: ohara-manager/client/src/context/WorkspaceContext.js [hunk 2]
```diff
@@ -39,12 +39,19 @@ const WorkspaceProvider = ({ children }) => {
     a.settings.name.localeCompare(b.settings.name),
   );
 
+  const findByWorkspaceName = workspaceName => {
```
Comment (konekoya) [inline ohara-manager/client/src/context/WorkspaceContext.js]: @saivirtue This method returns the workspace data which matches the given workspaceName by the caller :)

Comment (konekoya) [reply ohara-manager/client/src/context/WorkspaceContext.js]: FYI: if `[].find` couldn't find the target item from a given Array. It returns an `undefined` instead

Comment (saivirtue) [reply ohara-manager/client/src/context/WorkspaceContext.js]: Thanks for the explanation :)

File: ohara-manager/client/src/AppProviders.js [hunk 3]
```diff
@@ -28,6 +28,7 @@ import { WorkspaceProvider } from './context/WorkspaceContext';
 import { PipelineProvider } from './context/PipelineContext';
 import { NewWorkspaceProvider } from './context/NewWorkspaceContext';
 import { NodeDialogProvider } from './context/NodeDialogContext';
+import { TopicProvider } from './context/TopicContext';
```
Comment (konekoya) [inline ohara-manager/client/src/AppProviders.js]: Add a new provider

File: ohara-manager/client/src/components/Pipeline/Toolbar.js [hunk 4]
```diff
@@ -56,11 +56,11 @@ const Toolbar = props => {
...
       <div className="toolbox-toggle">
         <ButtonGroup variant="contained" color="default" size="small">
-          <IconButton onClick={() => handleClick('topic')}>
+          <IconButton onClick={() => handleClick('source')}>
             <FlightTakeoffIcon />
           </IconButton>
-          <IconButton onClick={() => handleClick('source')}>
-            <DnsIcon />
+          <IconButton onClick={() => handleClick('topic')}>
+            <StorageIcon />
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Toolbar.js]: Couldn't figure out a better icon for this...

File: ohara-manager/client/src/components/common/Dialog/DrabblePaper.js [hunk 5]
```diff
@@ -21,7 +21,10 @@ import Draggable from 'react-draggable';
 
 const DrabblePaper = props => {
   return (
-    <Draggable cancel={'[class*="MuiDialogContent-root"]'}>
+    <Draggable
+      handle=".MuiDialogTitle-root"
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Dialog/DrabblePaper.js]: This prop ensures that only the title is draggable

**Pull request merged**